### PR TITLE
DOCSP-37523 - Connect to MongoDB

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37523
Staging - https://preview-mongodbmongokart.gatsbyjs.io/pymongo/docsp-37523-connect/connect/connect-to-mongodb/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/connect/connect-to-mongodb.txt [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+.. tip::
+
+   Follow the :atlas:`Atlas driver connection guide </driver-connection?tck=docs_driver_python>`
+   to retrieve your connection string.
+
+When you connect to Atlas, we recommend using the {+stable-api+} client option to avoid
+breaking changes when Atlas upgrades to a new version of MongoDB Server.
+To learn more about the {+stable-api+} feature, see the :ref:`{+stable-api+} page
+<pymongo-stable-api>`.
```
Comment (mongoKart) [inline source/connect/connect-to-mongodb.txt:113]: Link will be broken until page is created

File: source/connect/connect-to-mongodb.txt [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+       these options.
+
+To use a connection URI, pass it as a string to the ``MongoClient`` constructor. In the 
+following example, the driver uses a sample connection URI to connect to a MongoDB 
+instance on port ``27017`` of ``localhost``: 
+
+.. literalinclude:: /includes/connect/local_connection.py
+   :language: python
+   :start-after: # start local connection
+   :end-before: # end local connection
```
Comment (mongoKart) [inline source/connect/connect-to-mongodb.txt:76]: Code examples will be added to all pages at the end of migration

File: source/connect/connect-to-mongodb.txt [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+   :keywords: connection string, URI, server, Atlas, settings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide shows you how to connect to a MongoDB instance or replica set
+deployment using {+driver-short+}.
```
Comment (jordan-smith721) [inline source/connect/connect-to-mongodb.txt:21]:
```suggestion
This guide shows you how to use {+driver-short+} to connect to a MongoDB instance or replica set
deployment.
```

File: source/connect/connect-to-mongodb.txt [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+   Because each ``MongoClient`` object represents a pool of connections to the
+   database, most applications require only a single instance of
+   ``MongoClient``, even across multiple requests. However, if you fork
+   a process, the child process *does* need its own ``MongoClient`` object.
+   To learn more, see the :ref:`FAQ <pymongo-faq>` page.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string?tck=docs_driver_python>` in the
+MongoDB Server documentation.
```
Comment (jordan-smith721) [inline source/connect/connect-to-mongodb.txt:88]: For consistency

```suggestion
MongoDB Server manual.
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM + two small things

File: source/connect/connect-to-mongodb.txt [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password@``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
```
Comment (NoahStapp) [inline source/connect/connect-to-mongodb.txt:48]: > authSource

What is `authSource` referring to here? I don't see any references to it besides the two in this breakdown block.

**Review by NoahStapp — CHANGES_REQUESTED** [review]

File: source/connect/connect-to-mongodb.txt (hunk 5 above)
Comment (mongoKart) [reply source/connect/connect-to-mongodb.txt:48]: added a link to the auth page.

**Review by NoahStapp — APPROVED** [review]

**Pull request merged**